Rethinking_2/Chp_06.ipynb

```diff
@@ -0,0 +1,3075 @@
+{
```

Comment:
<p>Forgot the ";" at the end of the line ;)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>